services/blockchain-indexer/shared/dataService/business/transactionsEstimateFees.js

```diff
@@ -0,0 +1,118 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2023 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+const { HTTP } = require('lisk-service-framework');
+
+const { isMainchain, resolveMainchainServiceURL } = require('./interoperability');
+const { dryRunTransactions } = require('./transactionsDryRun');
+const { tokenHasUserAccount, getTokenConstants } = require('./token');
+
+const { MODULE, COMMAND, EVENT } = require('../../constants');
+const regex = require('../../regex');
+const { parseToJSONCompatObj } = require('../../utils/parser');
+const { requestConnector, requestFeeEstimator } = require('../../utils/request');
+
+const resolveChannelInfo = async (chainID) => {
+	if ((await isMainchain()) && !regex.MAINCHAIN_ID.test(chainID)) {
+		// Redirect call to the mainchain service
+		const serviceURL = await resolveMainchainServiceURL();
+		const invokeEndpoint = `${serviceURL}/api/v3/invoke`;
+		const channelInfo = await HTTP.post(invokeEndpoint, { endpoint: 'interoperability_getChannel', params: { chainID } });
+		return channelInfo;
```

Comment:
Please verify the following.
```suggestion
		const { data: { data: channelInfo } } = await HTTP.post(invokeEndpoint, { endpoint: 'interoperability_getChannel', params: { chainID } });
		return channelInfo;
```